README.md

```diff
@@ -1,2 +1,54 @@
-# hear-baseline
-Simple baseline model for the HEAR 2021 NeurIPS competition
+![HEAR2021](https://neuralaudio.ai/assets/img/hear-header-sponsor.jpg)
+# HEAR 2021 Baseline
+
+A simple DSP-based audio embedding consisting of a Mel-frequency spectrogram followed
+by a random projection. Serves as the baseline model for the HEAR 2021 and implements
```

Comment:
naive baseline? Can we call the repo a naive baseline too?

Reply:
Sure we can call this baseline the naive baseline. I'm a bit worried about the pip install / module name being too wordy. ```pip install hearnaivebasline`` is a bit awkward. What about this: if we create a conv baseline, which would be cool, can we include it in this repo? we could have `hearbaseline.naive` and `hearbaseline.conv`?